requirements.txt

```diff
@@ -6,7 +6,6 @@ matplotlib
 numpy
 pandas
 pillow
-pyqt5<5.11
```

Comment:
Wasn't it used before?

Reply:
It is still used, but in the current configuration attempting to pip-install PyXRF into environment where PyXRF was already installed using conda messes up the environment. So currently there will be a 3-rd step in the instructions for pip-installing PyXRF:
- install scikit-beam (from nsls2forge)
- pip-install pyqt5 (if it is not installed)
- pip-install PyXRF
For develop install:
- install PyXRF from nsls2forge
- $pip install -e . (from the local PyXRF directory)
Actually only develop install is currently used and needed. Recommended way to install PyXRF is to use nsls2forge (pip install is only for experts).

Reply:
Sounds good.